src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -558,6 +559,364 @@ def __init__(self, launch_string):
         super().__init__(details)
 
 
+def _get_fluent_launcher_mode(start_container, container_dict):
+    """Get Fluent launch mode."""
+    if pypim.is_configured():
+        fluent_launch_mode = LaunchMode.PIM
+    elif start_container is True or (
+        start_container is None
+        and (container_dict or os.getenv("PYFLUENT_LAUNCH_CONTAINER") == "1")
+    ):
+        if check_docker_support():
+            fluent_launch_mode = LaunchMode.CONTAINER
+        else:
+            raise DockerContainerLaunchNotSupported()
+    else:
+        fluent_launch_mode = LaunchMode.STANDALONE
+
+    del start_container
```

Comment:
This `del` was being applied at the outer scope. This doesn't affect the outer variable.

Reply:
We will include this change in next PR. Thank you.